pkg/broker/filter/filter_handler.go

```diff
@@ -305,16 +305,17 @@ func (r *Handler) filterEventByAttributes(ctx context.Context, attrs map[string]
 	// exactly the same as the attributes defined in the current version of the
 	// CloudEvents spec.
 	ce := map[string]interface{}{
-		"specversion":         event.SpecVersion(),
-		"type":                event.Type(),
-		"source":              event.Source(),
-		"subject":             event.Subject(),
-		"id":                  event.ID(),
-		"time":                event.Time().String(),
-		"schemaurl":           event.SchemaURL(),
-		"datacontenttype":     event.DataContentType(),
-		"datamediatype":       event.DataMediaType(),
-		"datacontentencoding": event.DataContentEncoding(),
+		"specversion":     event.SpecVersion(),
+		"type":            event.Type(),
+		"source":          event.Source(),
+		"subject":         event.Subject(),
+		"id":              event.ID(),
+		"time":            event.Time().String(),
+		"schemaurl":       event.DataSchema(),
+		"datacontenttype": event.DataContentType(),
+		"datamediatype":   event.DataMediaType(),
```

Comment:
`datamediatype` is not real, it comes from the datacontenttype, it is just the parsed version of that field without char encoding and other extensions.

Reply:
so, remove?

Reply:
it will be ignored by everything, it does no harm, but it is not used